src/Microsoft.Health.Fhir.Azure/FhirServerBuilderAzureRegistrationExtensions.cs

```diff
@@ -77,5 +79,36 @@ public static IFhirServerBuilder AddContainerRegistryTokenProvider(this IFhirSer
 
             return fhirServerBuilder;
         }
+
+        public static IFhirServerBuilder AddAzureIntegrationDataStoreClient(this IFhirServerBuilder fhirServerBuilder, IConfiguration configuration)
```

Comment:
>public static IFhirServerBuilder AddAzureIntegrationDataStoreClient(this IFhirServerBuilder fhirServerBuilder, IConfiguration configuration) [](http://example.com/codeflow?start=8&length=140)

Can we add a comment for this to make it clearer. Looks like its adding Blob storage clients?

Reply:
Sure, added. For more details, we would like to reuse storage as export, but now it is at export operation level. To keep backward, we added a new blob store at common level, after we enable this on Paas, we would like to unify the usage for integration store.